packages/terra-functional-testing/src/services/wdio-terra-service.js

```diff
@@ -2,18 +2,58 @@
 const expect = require('expect');
 const { accessibility } = require('../commands/validates');
 const { toBeAccessible } = require('../commands/expect');
+const viewportHelpers = require('../commands/viewport-helpers');
+const hideInputCaret = require('../commands/hide-input-caret');
 
 class TerraService {
   /**
    * Service hook executed prior to test execution.
    * Initializes the Terra Service's custom commands.
    */
-  before() {
+  before(capabilities) {
+    /* Add the Jest expect module the use the Jest matchers. */
     global.expect = expect;
     global.expect.extend({ toBeAccessible });
 
-    global.Terra = {};
-    global.Terra.validates = { accessibility };
+    /* Add a Terra global with access to Mocha-Chai test helpers. */
+    global.Terra = {
+      validates: { accessibility },
+
+      /* `viewports` provides access Terra's list of test viewports. */
+      viewports: viewportHelpers.getViewports,
+
+      /* `describeViewports` provides a custom Mocha `describe` block for looping test viewports. */
+      describeViewports: viewportHelpers.describeViewports,
+
+      /* `hideInputCaret` hides the blinking input caret that appears in inputs or editable text areas. */
+      hideInputCaret,
+    };
+
+    /* IE driver takes a longer to be ready for browser interactions. */
+    if (capabilities.browserName === 'internet explorer') {
+      global.browser.pause(10000);
```

Comment:
Does `waitForExist` work at this point in the service?

https://webdriver.io/docs/api/element/waitForExist.html

The pause documentation sort of applies to this situation, the selector could potentially be something on the page (html/body tag maybe?)

https://webdriver.io/docs/api/browser/pause.html

> Pauses execution for a specific amount of time. It is recommended to not use this command to wait for an element to show up. In order to avoid flaky test results it is better to use commands like waitForExist or other waitFor* commands.

Reply:
`waitForExist` requires a selector and there isn't a good selector to use here. Looking back at #183 that added this 10 seconds was the suggested wait time.